### PR TITLE
Added installation instructions to the readme

### PR DESCRIPTION
**Pull request by UdattaChowdhury**
I have added the git commands to install all packages.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,15 +63,15 @@ https://nuclearmusic.rtfd.io/
...
-| GURU (Gentoo)  | https://github.com/gentoo/guru/tree/master/media-sound/nuclear-bin | [scardracs](https://github.com/scardracs)    |
-| Homebrew (Mac) | https://formulae.brew.sh/cask/nuclear                              | Homebrew                                     |
-| Snap           | https://snapcraft.io/nuclear                                       | [nukeop](https://github.com/nukeop)          |
-| Flatpak        | https://flathub.org/apps/details/org.js.nuclear.Nuclear            | [nukeop](https://github.com/nukeop)          |
+| Package type   | Link                                                               | Maintainer                                   | Installation Method                           |
+|:--------------:|:------------------------------------------------------------------:|:--------------------------------------------:|:---------------------------------------------:|
+| AUR (Arch)     | https://aur.archlinux.org/packages/nuclear-player-bin/             | [nukeop](https://github.com/nukeop)          |yay -s nuclear-player-git                      |
```
Comment (nukeop) [inline README.md:68]: This line is wrong

Comment (UdattaChowdhury) [reply README.md:68]: Correcting it right away

**Comment by UdattaChowdhury** [comment]
Is it now correct?

**Comment by UdattaChowdhury** [comment]
I am creating a fresh pull request.

**Pull request closed without merging**